raster/r.kappa/calc_metrics.c

```diff
@@ -213,3 +242,20 @@ static int longcomp(const void *aa, const void *bb)
 
     return (*a > *b);
 }
+
+/* Implements improved Kahan–Babuska algorithm by Neumaier, A. 1974 */
+void update_sum(double *sum, double *c, double value)
+{
+    double tmp;
+
+    tmp = *sum + value;
```

Comment:
Elsewhere we declare and init at the same line.

Reply:
Merged to a single line.